src/scripts/quiz.js

```diff
@@ -0,0 +1,13 @@
+var a = document.querySelector(".quiz-cl");
+
+let b = document.querySelector(".btnText");
+
+let c = document.querySelector('.bt2');
+
+b.addEventListener('click', ab());
+
+
+let ab = () =>{
```

Comment:
Please give a proper name to this function. As i cannot understand what It will do by its name

Reply:
it's an unnecessary file I forgot to delete it.